TOC-tidb-cloud.md

```diff
@@ -64,11 +111,240 @@
+      - [`SHOW DATABASES`](/sql-statements/sql-statement-show-databases.md)
+      - [`SHOW DRAINER STATUS`](/sql-statements/sql-statement-show-drainer-status.md)
+      - [`SHOW ENGINES`](/sql-statements/sql-statement-show-engines.md)
+      - [`SHOW ERRORS`](/sql-statements/sql-statement-show-errors.md)
+      - [`SHOW [FULL] FIELDS FROM`](/sql-statements/sql-statement-show-fields-from.md)
+      - [`SHOW GRANTS`](/sql-statements/sql-statement-show-grants.md)
+      - [`SHOW INDEX [FROM|IN]`](/sql-statements/sql-statement-show-index.md)
+      - [`SHOW INDEXES [FROM|IN]`](/sql-statements/sql-statement-show-indexes.md)
+      - [`SHOW KEYS [FROM|IN]`](/sql-statements/sql-statement-show-keys.md)
+      - [`SHOW MASTER STATUS`](/sql-statements/sql-statement-show-master-status.md)
+      - [`SHOW PLUGINS`](/sql-statements/sql-statement-show-plugins.md)
+      - [`SHOW PRIVILEGES`](/sql-statements/sql-statement-show-privileges.md)
+      - [`SHOW [FULL] PROCESSSLIST`](/sql-statements/sql-statement-show-processlist.md)
+      - [`SHOW PROFILES`](/sql-statements/sql-statement-show-profiles.md)
+      - [`SHOW PUMP STATUS`](/sql-statements/sql-statement-show-pump-status.md)
+      - [`SHOW SCHEMAS`](/sql-statements/sql-statement-show-schemas.md)
+      - [`SHOW STATS_HEALTHY`](/sql-statements/sql-statement-show-stats-healthy.md)
+      - [`SHOW STATS_HISTOGRAMS`](/sql-statements/sql-statement-show-histograms.md)
+      - [`SHOW STATS_META`](/sql-statements/sql-statement-show-stats-meta.md)
+      - [`SHOW STATUS`](/sql-statements/sql-statement-show-status.md)
+      - [`SHOW TABLE NEXT_ROW_ID`](/sql-statements/sql-statement-show-table-next-rowid.md)
+      - [`SHOW TABLE REGIONS`](/sql-statements/sql-statement-show-table-regions.md)
+      - [`SHOW TABLE STATUS`](/sql-statements/sql-statement-show-table-status.md)
+      - [`SHOW [FULL] TABLES`](/sql-statements/sql-statement-show-tables.md)
+      - [`SHOW [GLOBAL|SESSION] VARIABLES`](/sql-statements/sql-statement-show-variables.md)
+      - [`SHOW WARNINGS`](/sql-statements/sql-statement-show-warnings.md)
+      - [`SHUTDOWN`](/sql-statements/sql-statement-shutdown.md)
+      - [`SPLIT REGION`](/sql-statements/sql-statement-split-region.md)
+      - [`START TRANSACTION`](/sql-statements/sql-statement-start-transaction.md)
+      - [`TABLE`](/sql-statements/sql-statement-table.md)
+      - [`TRACE`](/sql-statements/sql-statement-trace.md)
+      - [`TRUNCATE`](/sql-statements/sql-statement-truncate.md)
+      - [`UPDATE`](/sql-statements/sql-statement-update.md)
+      - [`USE`](/sql-statements/sql-statement-use.md)
+      - [`WITH`](/sql-statements/sql-statement-with.md)
+    - Data Types
+      - [Overview](/data-type-overview.md)
+      - [Default Values](/data-type-default-values.md)
+      - [Numeric Types](/data-type-numeric.md)
+      - [Date and Time Types](/data-type-date-and-time.md)
+      - [String Types](/data-type-string.md)
+      - [JSON Type](/data-type-json.md)
```

Comment:
This is still experimental. FYI: https://docs.pingcap.com/tidb/dev/data-type-json

Reply:
Since this PR will not affect TiDB Cloud TOC. Let us recheck this comment and review the TOC after picking the changes to the v6.1 branch.